src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -29,6 +30,7 @@
 
     private Stage primaryStage;
     private Logic logic;
+    private GuiState guiState = GuiState.SUMMARY;
```

Comment:
yup as u mentioned i think moving to constructor wld be better! since default vals usually are immediately declared only for static variables